### PR TITLE
agglomerateByPrevalence

### PR DESCRIPTION
**Pull request by FelixErnst**

### CODE REVIEW

**Comment by FelixErnst** [comment]
@tvborm I added you as reviewer, so that you can also have a look at a PR from that side 😉

File: NAMESPACE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Generated by roxygen2: do not edit by hand
 
 export(TAXONOMY_RANKS)
```
Comment (TuomasBorman) [inline NAMESPACE:3]: @FelixErnst ok, that's nice. I haven't reviewed before so I hope this test comment does not bother

**Review by antagomir — APPROVED** [review]
Seems fine.

**Comment by TuomasBorman** [comment]
I got an error while trying to run the function. Though, I am not sure, if it is related to the functionality itself

Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function ‘agglomerateByRank’ for signature ‘"missing"’

**Comment by FelixErnst** [comment]
> I got an error while trying to run the function. Though, I am not sure, if it is related to the functionality itself
> 
> Error in (function (classes, fdef, mtable) :
> unable to find an inherited method for function ‘agglomerateByRank’ for signature ‘"missing"’

Should be fixed now

**Comment by TuomasBorman** [comment]
> > I got an error while trying to run the function. Though, I am not sure, if it is related to the functionality itself
> > Error in (function (classes, fdef, mtable) :
> > unable to find an inherited method for function ‘agglomerateByRank’ for signature ‘"missing"’
> 
> Should be fixed now

Ok, now it is working

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**